llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -407,6 +407,54 @@ def SYNTACORE_SCR7 : RISCVProcessorModel<"syntacore-scr7",
+                                              [Feature64Bit,
+                                               FeatureStdExtI,
+                                               FeatureStdExtZifencei,
+                                               FeatureStdExtZicsr,
+                                               FeatureStdExtZicntr,
+                                               FeatureStdExtZihpm,
+                                               FeatureStdExtZihintpause,
+                                               FeatureStdExtM,
+                                               FeatureStdExtA,
+                                               FeatureStdExtF,
+                                               FeatureStdExtD,
+                                               FeatureStdExtC,
+                                               FeatureStdExtV,
+                                               FeatureStdExtZvl256b,
+                                               FeatureStdExtZfh,
+                                               FeatureStdExtZvfh,
+                                               FeatureStdExtZba,
+                                               FeatureStdExtZbb,
+                                               FeatureStdExtZbs,
+                                               FeatureStdExtZicbom,
+                                               FeatureStdExtZicbop,
+                                               FeatureStdExtZicboz,
+                                               FeatureStdExtH,
+                                               FeatureStdExtZihintntl,
+                                               FeatureStdExtZfhmin,
+                                               FeatureStdExtZfa,
+                                               FeatureStdExtZkt,
+                                               FeatureStdExtZcb,
+                                               FeatureStdExtZvbb,
+                                               FeatureStdExtZvbc,
+                                               FeatureStdExtZawrs,
+                                               FeatureStdExtZvkng,
+                                               FeatureStdExtZicond,
+                                               FeatureUnalignedScalarMem,
+                                               FeatureUnalignedVectorMem,
+                                               FeatureStdExtSvnapot,
+                                               FeatureStdExtSvpbmt,
+                                               FeatureStdExtSvinval,
+                                               FeatureStdExtZfbfmin,
+                                               FeatureStdExtZvfbfmin,
+                                               FeatureStdExtZvfbfwma],
+                                              [TuneNoDefaultUnroll,
+                                               TuneOptimizedZeroStrideLoad,
+                                               TuneNoSinkSplatOperands,
```

Comment:
I'm surprised to see `TuneNoSinkSplatOperands` is enabled.
Does this match the microarchitecture? The design of scalar-to-vector buffer is a common design, but I was thinking this may have been fixed in some cores.

Reply:
Yes, it was, my bad.

Reply:
Addressed